README.md

```diff
@@ -82,7 +82,10 @@ The 7 possible "type" options are:
 
 
 ```bash
-$ /target/debug/itunesdb_parser <path-to-itunesdb-file> '<type>'
+$ cd iTunesDB-Parser
```

Comment:
1. the path to build in is the `parser` directory because thats where the code is
2. No need to run `cargo init` that is for new projects

Reply:
This path is already part of what is cloned, there's no need to include it here. You only need to change directory into `parser` to do the build

Reply:
so it should be cd into parser, then cargo build, then run the target binary